### PR TITLE
Create tries

### PR DESCRIPTION
**Pull request by nishant4500**
## 📥 Pull Request

Important content for day-other as trie data structure has been added.

Fixes # (issue number)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
You need to write your file name as `tries.md`. You can then choose a topic with a structured format from our folders and files. The file structure is located at https://ajay-dhangar.github.io/algo/dsa-roadmap/.

File: docs/tries.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+     l
+     |
+     e 
+     | 
+     (word end)
+    /
+   p 
+   |
+   (word end)
+```
```
Comment (ajay-dhangar) [inline docs/tries.md]: Try Mermaid Graph for better understanding and good UI/UX. It follows best practices, and you can check out https://mermaid.js.org/ for reference.

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by nishant4500** [comment]
@ajay-dhangar Hey sir could you please merged now i think now i made all the changes with new UI

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**